controller/src/main.cpp

```diff
@@ -98,7 +98,8 @@ static void DEV_MODE_comms_handler() {
   Serial.print(',');
   Serial.print(controller_status.sensor_readings.flow_ml_per_min / 1000.0f);
   Serial.print(',');
-  Serial.print(controller_status.sensor_readings.volume_ml);
+  // Report volume in cL because because that fits on the graph better.
+  Serial.print(controller_status.sensor_readings.volume_ml / 10.f);
```

Comment:
If you want to change the graph range, it's defined here https://github.com/RespiraWorks/VentilatorSoftware/blob/master/gui/ScopeView.qml#L69 though it's common for all three graphs - it obviously shouldn't be; lmk what ranges are appropriate for each and I'll make them different.

Reply:
In the GUI the graphs are on different axes, so it's not as important.  But I also think this should be L/min, I just messed it up when I defined it.  e.g. squint at https://www.hamilton-medical.com/en_US/Solutions/Ventilation-Cockpit-user-interface.html